screens/EditActivity/index.tsx

```diff
@@ -93,9 +93,12 @@ const styles = StyleSheet.create({
   },
   inputContainer: {
     flex: 1,
+    paddingVertical: 0,
```

Comment:
this is not doing anything

Reply:
The input has a vertical padding that this is overriding. I did this to align the input in this form with the "Done" button
<img width="365" alt="image" src="https://github.com/user-attachments/assets/e896753b-594f-40a6-908c-3ac8b43ba722" />